package.json

```diff
@@ -21,11 +21,8 @@
     "url": "https://github.com/biancadanforth/tracking-protection-shield-study/issues"
   },
   "devDependencies": {
-    "addons-linter": "^0.28.2",
```

Comment:
This seems currently unused.
There is a `lint:addons-linter` npm script, but didn't seem to do anything or run the addons-linter:

https://github.com/biancadanforth/tracking-protection-shield-study/blob/de4df346b233adbb74a2a51234ffc00ab867f816/package.json#L69

I believe the other 2 extraneous dependencies, **clipboardy** and **doctoc**, came from the shield template since I couldn't find any usages via <kbd>git grep clipboardy</kbd> or <kbd>git grep doctoc</kbd>.

Reply:
In the template repo, the command for `"lint:addons-linter"` is `"addons-linter addon/webextension/"`, but we can just leave it out for now, since this is a legacy add-on and we're doing all sorts of crazy and weird stuff.